src/gc.c

```diff
@@ -785,21 +785,21 @@ static inline int maybe_collect(void)
 
 // preserved values
 
-DLLEXPORT int jl_gc_n_preserved_values(void)
+int jl_gc_n_preserved_values(void)
```

Comment:
At least I think they are potentially useful and shouldn't cause any issues to be exported.